Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -192,15 +195,25 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         Task {
             let captureResult = try? await DemoMerchantAPI.sharedService.caputureOrder(orderID: result.orderID)
             let status = captureResult?.status ?? ""
-            updateTitle("Order ID:\(result.orderID) status: \(status)")
+            let vault = captureResult?.paymentSource?.card.attributes?.vault
+            let orderInfo = "Order ID: \(result.orderID) \n Status: \(status)"
+            let vaultInfo = vault != nil ? "Vault Status: \((vault?.status) ?? "") \n Vault ID: \((vault?.id) ?? "")" : ""
+            let customerInfo = vault != nil ? "Customer ID: \((vault?.customer.id ?? ""))" : ""
+            
+            updateTitle("\(orderInfo) \n \(vaultInfo)\n \(customerInfo)")
```

Comment:
This block of code is duplicated b/w the capture & authorize functions. We can consolidate this into a helper function so we don't have to duplicate. Maybe something like `displayResult(orderID: String, vaultID: String, customerID: String)`.

Reply:
Addressed in commit 6057b4e6155b57c8c2fa4f361c0410eb0f29f1e3